src/transformed/transformed-dark.json

```diff
@@ -807,7 +807,7 @@
           "y": 4,
           "blur": 16,
           "spread": 0,
-          "color": "#000000 16%",
+          "color": "#00000052",
```

Comment:
This...seems wrong? Seems to be missing the rgba, slash appending it to the end

Reply:
We have these transforming to 8 digit HEX codes (##RRGGBBAA):

![Screenshot 2023-09-12 at 3 58 40 PM](https://github.com/Fullscript/aviary-tokens/assets/49457972/1f435f44-8c24-42e0-9204-d264e61869e1)

Reply:
Ooooh, wow I didnt even realize this is a thing! TIL